PWGHF/TableProducer/candidateCreator2Prong.cxx

```diff
@@ -85,10 +83,8 @@ struct HfCandidateCreator2Prong {
     runNumber = 0;
   }
 
-  void process(aod::Collisions const& collisions,
-               soa::Join<aod::Hf2Prongs, aod::HfPvRefit2Prong> const& rowsTrackIndexProng2,
-               aod::BigTracks const& tracks,
-               aod::BCsWithTimestamps const& bcWithTimeStamps)
+  template <bool doPvRefit = false, typename Cand>
+  void runCreator2Prong(aod::Collisions const& collisions, Cand const& rowsTrackIndexProng2, aod::BigTracks const& tracks, aod::BCsWithTimestamps const& bcWithTimeStamps)
```

Comment:
Can you keep the line breaks with one argument per line, for clarity?

Reply:
Done